.github/workflows/flannel.yml

```diff
@@ -0,0 +1,17 @@
+name: flannel-images
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
```

Comment:
I don't think we will want this one on a schedule since flannel doesn't need updates every version. We can publish a new version with a PR merges (push to master in github workflow terms) and when we cut a release?

https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#push 
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#release

Reply:
We can tag the image version as pre release for merge to master and use the git tag for the release.

Reply:
So, how I can to resolve this moment? Do I need to remove `.github/workflows/flannel.yml`?

Reply:
use the docs above to add pr and release triggers.  an example of building on PR merge:

```
on:
  push:
    branches:
      - master
```

Reply:
ok